lib/va_profile/models/telephone.rb

```diff
@@ -146,6 +146,32 @@ def self.build_from(body)
           effective_start_date: body['effective_start_date']
         )
       end
+
+      def self.transaction_response_class
+        VAProfile::ProfileInformation::TelephoneTransactionResponse
+      end
+
+      def self.transaction_status_path(user, transaction_id)
+        "#{user.vet360_id}/telephones/status/#{transaction_id}"
+      end
+
+      def self.send_change_notifcations?
+        true
+      end
+
+      def contact_info_attr
+        'telephone'
+      end
+
+      def self.contact_info_attr(record)
```

Comment:
This might look a little cleaner as a CASE statement

Reply:
Fixing now.